app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_or_the_parts_should_be_equal_to_value():
-    pass
+    assert sum(split_integer(6, 2)) == 6
+    assert sum(split_integer(17, 4)) == 17
+    assert sum(split_integer(32, 6)) == 32
 
 
 def test_should_split_into_equal_parts_when_value_is_divisible_by_parts():
-    pass
+    assert len(split_integer(6, 2)) == 2
+    assert len(split_integer(17, 4)) == 4
+    assert len(split_integer(32, 6)) == 6
 
 
 def test_should_return_part_equals_to_a_value_when_slitting_into_one_part():
-    pass
+    assert split_integer(8, 1) == [8]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal():
-    pass
+    result = split_integer(35, 6)
+    assert result == sorted(result)
```

Comment:
```suggestion
    assert split_integer(35, 6) == sorted(result)
```
do not create an extra variable when it makes code less readable

Reply:
ты можешь использовать конкретный ожидаемый результат: 
```
assert split_integer(35, 6) == [5, 6, 6, 6, 6, 6]
```

Reply:
Так же во всех тестах
Лучше писать конкретный ожидаемый результат, потому что: 

1. меньше шансов допустить ошибку при написании логики проверки
2. так проще читать тесты, потому что, например для теста `test_difference_between_max_and_min_number` ошибка будет не "False where False is `max(result) - min(result) <= 1`" а будет выведено ожидаемое значение и актуальное, которое вернула функция

Reply:
Спасибо за пояснение, и терпение.